ui/packages/catalog-ui-search/src/main/webapp/component/result-form/result-forms.view.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+/* global setTimeout */
+const SearchFormViews = require('component/search-form/search-form.view')
+
+module.exports = SearchFormViews.extend({
+  initialize: function () {
+    SearchFormViews.prototype.initialize.call(this)
+  },
+  changeView: function () {
```

Comment:
❓ I don't think it is actually even necessary to replicate/use this logic when dealing with result forms as opposed to search forms. 

Result forms have their own specific lightbox view they use for editing. Search forms rely on the existing query editing infrastructure to do their CRUD, so they need to set aspects of the underlying `Query.js` model that the query editing infrastructure relies on.

Pretty sure you could essentially just take all the logic that got added to `query-add.view`, move it in here, and cause that to trigger on `this.listenTo(this.model, 'change:type', this.showLightboxEditor)` or something. Not sure that it's entirely necessary to do now though. This is an `experimentalFeature`, after all.